prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -54,6 +54,11 @@
                   [showIfEmpty]="true"
                   (route)="onRoute(demographicRoutePath)"></app-address-view>
 
+<app-address-view title="Additional Addresses"
```

Comment:
address-view is a simple component that takes an address and displays it. 

What benefit do you get changing how that component functions, over using an NgFor in this component?

Reply:
Reverted changes to the `AddressViewComponent`, using `*ngFor` instead

Reply:
The reason I did that was because we can have multiple additional addresses under one title. Now using a combination of ngFor and app-subheader. Reverted the work on the `AddressViewComponent`